pkg/apis/compliance/v1alpha1/compliancescan_types.go

```diff
@@ -204,6 +205,10 @@ type ComplianceScanSettings struct {
 	// for the scanner container and 200Mi memory with 100m CPU for the api-resource-collector
 	// container).
 	ScanLimits map[corev1.ResourceName]resource.Quantity `json:"scanLimits,omitempty"`
+
+	// Timeout is the maximum amount of time the scan can run. If the scan
+	// hasn't finished by then, it will be aborted.
+	Timeout string `json:"timeout,omitempty"`
```

Comment:
As discussed yesterday on a call, this should use a kubebuiler default (https://book.kubebuilder.io/reference/markers/crd-validation.html)